api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			},
+			"put": {
+				"summary": "Create or update an identity",
+				"tags": [ "policy" ],
+				"parameters": [
+					{
+						"name": "labels",
+						"in": "body",
+						"required": true,
+						"schema": {
+							"$ref": "#/definitions/Labels"
+						}
+					},
+					{
+						"name": "container-id",
+						"in": "query",
+						"type": "string"
+					}
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success",
+                                                "schema": {
+							"$ref": "#/definitions/Identity"
+                                                }
+					},
+                                        "default": {
+                                                "description": "Unexpected error",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			},
+			"delete": {
+				"summary": "Delete identity",
+				"tags": [ "policy" ],
+				"parameters": [
+					{ "$ref": "#/parameters/identity-id" },
+					{ "$ref": "#/parameters/identity-hash" },
+					{
+						"name": "container-id",
+						"in": "query",
+						"required": true,
+						"type": "string"
+					}
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success"
+					},
+                                        "404": {
+                                                "description": "Identity not found",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			}
+		},
+                "/identity/max-id": {
+			"get": {
+				"summary": "Retrieve the maximum identity ID",
+				"tags": [ "policy" ],
+				"responses": {
+					"200": {
+                                                "description": "Success",
+                                                "schema": {
+							"type": "integer"
+                                                }
+					},
+                                        "default": {
+                                                "description": "Unexpected error",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			}
+		},
+                "/ipam": {
+			"get": {
+				"summary": "Allocate an IP address",
```

Comment:
I agree that `GET` - `DELETE` is an odd pair for allocate/release. `PUT` - `DELETE` makes more sense. Fixed.